.github/actions/copy-workflow-go/action.yml

```diff
@@ -31,6 +31,9 @@ runs:
             # Update the version in go.mod. This alone ensures there's a diff.
             go mod edit -go $TARGET_VERSION
 
+            # We tidy immediately because some updates require it
+            go mod tidy
+
```

Comment:
See https://github.com/protocol/.github/runs/5815591028?check_suite_focus=true for example